movement/plot.py

```diff
@@ -0,0 +1,114 @@
+"""Wrappers to plot movement data."""
+
+from pathlib import Path
+
+import xarray as xr
+from matplotlib import pyplot as plt
+
+DEFAULT_PLOTTING_ARGS = {
+    "s": 15,
+    "cmap": "viridis",
+    "marker": "o",
+    "alpha": 1.0,
+}
+
+
+def trajectory(
+    da: xr.DataArray,
+    keypoint: str | list[str],
+    individual: int | str = 0,
+    frame_path: None | Path = None,
+    title: str | None = None,
+    **kwargs,
+) -> plt.Figure:
+    """Plot trajectory.
+
+    This function plots the  trajectory of a specified keypoint or the midpoint
+    between two keypoints for a given individual. The individual can be
+    specified by their index or name. By default, the first individual
+    is selected.
```

Comment:
I wouldn't like to restrict the centroid feature to only two options. In many cases users might want to show the centroid of multiple keypoints places on the body. The could of course pre-compute that and then call `plot.trajectory()`, but the flexibility of the "keypoint" option here offers a convenient alternative.
```suggestion
    This function plots the  trajectory of a specified keypoint or the centroid
    of multiple keypoints for a given individual. The individual can be
    specified by their index or name. By default, the first individual
    is selected.
```